geometry_msgs/Pose pose
int64 dir_x
int64 dir_y
int64 dir_z

